src/status_im/contexts/keycard/migrate/success/view.cljs

```diff
@@ -26,6 +26,6 @@
        {:resize-mode :contain
         :source      (resources/get-image :keycard-migration-succeeded)}]]
      [quo/button
-      {:on-press        #(rf/dispatch [:logout])
+      {:on-press        #(rf/dispatch [:profile.logout/logout])
```

Comment:
`:profile/logout` seems sufficient and easily searchable. Is there a reason to use the longer keyword with the extra qualification in `profile.logout`?

Reply:
The reason is I'm considering `profile.logout` as the qualified namespace and I'm using it for all the logout related events, so I wanted to be consistent.


Do you think is better to just name it `profile/logout`?